.github/workflows/ci-checks.yml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - main
+      - next
```

Comment:
is `next` still going to be around?

Reply:
No, it'll be deleted, but eventually we'll have another `next`, so this'll save us a step next time. That reminds me that I do need to turn off the auto-merge job though in this PR.